### PR TITLE
Update ug v1.1

### LINKED ISSUE #14: Update the UG
Refer to docs/UserGuide.md

### PR DESCRIPTION
**Pull request by Jayne1010**
Closes #14

### CODE REVIEW

**Comment by drustanyjt** [comment]
It feels like #26 already has everything from this PR (since it's just one extra commit for DG). I think we can close this PR and just track changes on #26.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
   pageNav: 3
 ---
 
-# AB-3 User Guide
+# Clinic Mate User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Clinic Mate is a **desktop app for managing contacts in a clinic, optimized for use via a  **Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Clinic Mate can get manage your patients' contact faster than traditional GUI apps.
```
Comment (drustanyjt) [inline docs/UserGuide.md]: Should it be "Clinic Mate can ~~get~~ manage your patients..." in the last sentence?

**Review by drustanyjt — CHANGES_REQUESTED** [review]
Got a small typo otherwise lgtm.

File: docs/UserGuide.md (hunk 1 above)
Comment (Jayne1010) [reply docs/UserGuide.md]: yep, changed :), anymore?

Comment (drustanyjt) [reply docs/UserGuide.md]: image

I think there's an extra double asterisk behind desktop.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
   pageNav: 3
 ---
 
-# AB-3 User Guide
+# Clinic Mate User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Clinic Mate is a **desktop app for managing contacts in a clinic, optimized for use via a  **Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Clinic Mate can manage your patients' contact faster than traditional GUI apps.
```
Comment (drustanyjt) [inline docs/UserGuide.md]: The double asterisk is here before "desktop"

**Review by drustanyjt — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (Jayne1010) [reply docs/UserGuide.md]: sorry didn't see the review jn, changed

**Review by drustanyjt — APPROVED** [review]
lgtm!

**Pull request merged**